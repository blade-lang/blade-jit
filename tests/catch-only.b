catch e {
  echo 'Should not show anything'
}