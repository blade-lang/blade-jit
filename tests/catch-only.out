^^^
`catch` and `finally` are only valid in `try` context